integration-tests/src/test/java/oracle/kubernetes/operator/utils/Domain.java

```diff
@@ -1757,9 +1753,71 @@ public boolean accept(File dir, String name) {
 
     // Copy all App files to the admin pod
     TestUtils.copyAppFilesToPod(appLocationOnHost, appLocationInPod, adminServerPod, domainNS);
+    // Run the script to build WAR, EAR or JAR file and deploy the App in the admin pod
```

Comment:
this is getting to be a very large file, i think we should create a JIRA to split it up into smaller files and group the tests accordingly.  if there is some reason why they all need to be in the one file, maybe we should think about refactoring some of the code out

Reply:
moved my added method to TestUtils